framework/web/UrlManager.php

```diff
@@ -242,6 +242,12 @@ public function parseRequest($request)
 
             Yii::trace('No matching URL rules. Using default URL parsing logic.', __METHOD__);
 
+            // before processing pathInfo and suffix there should not be end multiple slashes in $pathInfo
```

Comment:
I'd like the following wording:

`Ensure, that $pathInfo does not end with more than one slash`